docs/documentation/setup/networking.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: "Networking"
+sidebarTitle: "Networking"
+description: "Network configuration details for Infisical Cloud"
+---
+
+## Overview
+
+When integrating your infrastructure with Infisical Cloud, you may need to configure network access controls. This page provides the IP addresses that Infisical uses to communicate with your services.
+
+## Egress IP Addresses
+
+Infisical Cloud operates from two regions: US and EU. Depending on your region and security requirements, you may need to whitelist specific IP addresses to allow Infisical to communicate with your services.
```

Comment:
i think we should say something about inbound connection to their network and to only allow infisical to communicate, they can add these ips for their ingress rules